crates/ruff_linter/Cargo.toml

```diff
@@ -71,6 +70,7 @@ unicode-width = { workspace = true }
 unicode_names2 = { workspace = true }
 unicode-normalization = { workspace = true }
 url = { workspace = true }
+once_cell = "1.20.2"
```

Comment:
Did you mean to get rid of this dependency? It looks like you just moved it in this file

Reply:
lol, Rust Rover must have inserted the dependency again :D

Reply:
That also explains why the error count in ruff_linter suddenly dropped from 20 to 0